tsconfig.base.json

```diff
@@ -8,8 +8,8 @@
     "declarationMap": true,
     "esModuleInterop": true,
     "incremental": true,
-    "module": "ESNext",
-    "moduleResolution": "node",
+    "module": "ES2022",
+    "moduleResolution": "Bundler",
```

Comment:
what does this do?

Reply:
See https://devblogs.microsoft.com/typescript/announcing-typescript-5-2/#module-and-moduleresolution-must-match-under-recent-node-js-settings